src/Bonsai.Scripting.Python/RuntimeManager.cs

```diff
@@ -78,22 +78,30 @@ static void Initialize(string path)
         {
             if (!PythonEngine.IsInitialized)
             {
-                if (string.IsNullOrEmpty(path))
-                {
-                    path = Environment.GetEnvironmentVariable("VIRTUAL_ENV", EnvironmentVariableTarget.Process);
-                    if (string.IsNullOrEmpty(path)) path = Environment.CurrentDirectory;
-                }
+                path = EnvironmentHelper.GetVirtualEnvironmentPath(path);
+                EnvironmentHelper.SetVirtualEnvironmentPath(path);
```

Comment:
It looks like both of these functions change environment variables before initializing the python runtime. Is it necessary to set `VIRTUAL_ENV` to correctly configure the environment on Linux? I thought this was only necessary to support environment activation when no specific path was provided.